.github/workflows/build.yml

```diff
@@ -210,7 +210,7 @@ jobs:
           if [ "$SERVICE_NAME" = "k8s-proxy" ]; then
```

Comment:
does the single `=` work in bash?

Reply:
Yes, that was successful in the last run in the `master` branch.